### ISSUE TITLE
pip installed ldm

### ISSUE THREAD

**Issue by gitihobo** [justinpinkney/stable-diffusion#86]
line 11, in <module>
    from ldm.models.diffusion.ddim import DDIMSampler
line 20
    print self.face_rec_model_path
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
SyntaxError: Missing parentheses in call to 'print'. Did you mean print(...)?

**Comment by normantaipei** [comment]
I meet the same problem in python 3, did anyone solve it ?

**Comment by NormanCorma** [comment]
Try
`
pip uninstall ldm
`
`
pip install diffusers==0.3.0
`
`
pip install transformers scipy ftfy
`
`
pip install omegaconf
`

**Comment by gitihobo** [comment]
nope. asks for ldm module, then No module named 'einops' ldm.py", line 20
    print self.face_rec_model_path
    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
SyntaxError: Missing parentheses in call to 'print'. Did you mean print(...)?

**Comment by impiyush** [comment]
so actually its trying to load the ldm from within the same repo. 

I also ran into this same problem, installed the latest ldm package via pip and then realized what the code was trying to do.

To resolve this, just append the path to your root dir. Add this code to the top of the gradio_image_mixer.py file
```
import sys
sys.path.append("~/stable-diffusion")
```
